conan/cli/commands/cache.py

```diff
@@ -48,3 +49,26 @@ def cache_path(conan_api: ConanAPI, parser, subparser, *args):
         else:
             raise ConanException(f"'--folder {args.folder}' requires a recipe reference")
     return path
+
+
+@conan_subcommand(formatters={"text": default_text_formatter})
```

Comment:
If the command has no output to console we don't need to have the text formatter.
```suggestion
@conan_subcommand()
```